TwilioVerifyDemo/TwilioVerifyDemo/Common/KeyChainAccessGroupHelper.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  KeyChainAccessGroupHelper.swift
+//  TwilioVerifyDemo
+//
+//  Created by Alejandro Orozco Builes on 7/01/22.
+//  Copyright © 2022 Twilio. All rights reserved.
+//
+
+import Foundation
+import Security
+
+public struct KeyChainAccessGroupInfo {
+  public var prefix: String
+  public var keyChainGroup: String
+  public var rawValue: String
+}
+
+open class KeyChainAccessGroupHelper {
+
+  public class func getAccessGroupInfo() -> KeyChainAccessGroupInfo? {
+    let query: [String:Any] = [
+      kSecClass as String : kSecClassGenericPassword,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/3d084cdf558fe2c3c223d0f0881d860cd43299b1/TwilioVerifyDemo/TwilioVerifyDemo/Common/KeyChainAccessGroupHelper.swift#L22"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>KeyChainAccessGroupHelper.swift:22</code></td>
    </tr>
  </tbody>
</table>